src/common/SPELLS_DRUID.js

```diff
@@ -642,6 +642,11 @@ export default {
     name: 'Celestial Alignment',
     icon: 'artifactability_balancedruid_newmoon',
   },
+  ED_BUFF: {    
```

Comment:
Please use the full name to avoid naming conflicts and make it clearer what this is.